website/content/docs/agent/wal-logstore/enable.mdx

```diff
@@ -7,30 +7,7 @@ description: >-
 
 # Enable the experimental WAL LogStore backend
```

Comment:
Seems we should stop calling this experimental here as well (also needed in title and description above, out of suggestion range)
```suggestion
# Enable the WAL LogStore backend
```